src/lib.rs

```diff
@@ -30,6 +30,12 @@ use bytes::{BufMut, Buf};
 #[cfg(windows)]
 use tokio_named_pipes::NamedPipe;
 
+#[cfg(windows)]
+use winapi::um::minwinbase::SECURITY_ATTRIBUTES;
+#[cfg(windows)]
+use std::ptr;
```

Comment:
used only once, right?

maybe be importing not worth it, let alone with platform attribute